source/digits_hits/include/GateJPETActor.hh

```diff
@@ -0,0 +1,197 @@
+	 * @ATTENTION!: If particleName is incorrect any data will be generated
+	 * @TIP: U can use more then once this function to add more particles to recored
+	 * */
+	void SetParticleEnabled(G4String particleName);
+
+	/**
+	 * Record data about particle trackID
+	 * @param: enableTrackID - set true if u want have data about this
+	 */
+	void SetTrackIDEnabled(bool enableTrackID);
+
+	/**
+	 * Record data about particle eventID
+	 * @param: enableEventID - set true if u want have data abut this
+	 */
+	void SetEventIDEnabled(bool enableEventID);
+
+	/**
+	 * Record data about particle ParentID
+	 * @param: enableParentID - set true if u want have data abut this
+	 */
+	void SetParentIDEnabled(bool enableParentID);
+
+	/**
+	 * Record data about particle RunID
+	 * @pram: enableRunID - set true if u want have data abut this
+	 */
+	void SetRunIDEnabled(bool enableRunID);
+
+	/**
+	 * Record data about particle kinetic energy Ekin
+	 * @param: enableEkin - set true if u want have data abut this
+	 */
+	void SetEkinEnabled(bool enableEkin);
+
+	/**
+	 * Record data about partciel interaction time
+	 * @param: enableInteractionTime - set true if u want have data abut this
+	 */
+	void SetInteractionTimeEnabled(bool enableInteractionTime);
+
+	/**
+	 * Record data about Process Name witch conected is with particle
+	 * @param: enableProcessName - set true if u want have data abut this
+	 */
+	void SetProcessNameEnabled(bool enableProcessName);
+
+	/**
+	 * Record data about emission point
+	 * @param: enableEmissionPoint - set true if u want have data abut this
+	 */
+	void SetEmissionPointEnabled(bool enableEmissionPoint);
+
+	/**
+	 * REcord data about primary energy
+	 * @param: enablePrimaryEnergy - set true if u want have data abut this
+	 */
+	void SetPrimaryEnergy(bool enablePrimaryEnergy);
+
+protected:
+	GateJPETActor(G4String name, G4int depth=0);
+
+	//Setting values showing which data record and which not - ATTENTION!: In defaul mode all bool setting are FALSE and vector is EMPTY
+	/*Record data about x-position*/
+	bool EnableXPosition;
+	/*Record data about y-position*/
+	bool EnableYPosition;
+	/*Record data about z-position*/
+	bool EnableZPosition;
+	/*Record data about particle name*/
+	bool EnableParticleName;
+	/*Record data for particles from THIS list. ATTENTION: If this vector is empty class will be record data for every particles which fullfil conditions*/
+	std::vector<G4String> ParticleNames;
+	/*Record track ID*/
+	bool EnableTrackID;
+	/*Record Event ID*/
+	bool EnableEventID;
+	/*Record Parent ID*/
+	bool EnableParentID;
+	/*Record Run ID*/
+	bool EnableRunID;
+	/*Record Kinetic energy of particle*/
+	bool EnableEkin;
+	/*Record interaction time*/
+	bool EnableInteractionTime;
+	/*Record process name*/
+	bool EnableProcessName;
+	/*Record emission point*/
+	bool EnableEmissionPoint;
+	/*Record primary energy*/
+	bool EnablePrimaryEnergy;
+
+	//Values
+	double positionX;
```

Comment:
The convention in GATE for members of any class: mKineticEnergy 
or mEnablePrimaryEnergy;